source/ui/BioXAS/BioXASMainXASScanConfigurationView.cpp

```diff
@@ -232,3 +241,14 @@ void BioXASMainXASScanConfigurationView::onEdgeChanged()
 	if (energy_->value() != configuration_->energy())
 		energy_->setValue(configuration_->energy());
 }
+
+void BioXASMainXASScanConfigurationView::onEstimatedTimeChanged()
+{
+    configuration_->blockSignals(true);
```

Comment:
I can only find a block/unblock on the config file in IDEAS/REIXS.  I'll remove that part.